src/exception/filter/executor.ts

```diff
@@ -27,7 +28,7 @@ export class FilterExecutor {
 		exceptionFilter: ExceptionFilter,
 		context: HttpContext,
 		error: unknown,
-	): Response | undefined {
+	): Response | undefined | { topic: string; data: unknown } {
```

Comment:
minor, we can fix it later

```suggestion
	): Response | undefined | WebSocketPayload {
```